scripts/copy-models/index.js

```diff
@@ -2,16 +2,50 @@
 const yargs = require("yargs");
 
 const { promises: fsPromises } = require("fs");
-const { join } = require("path");
-const { spawnProcess } = require("../utils/spawn-process");
+const { join, resolve } = require("path");
 
-const { models } = yargs(process.argv.slice(2))
+const { models, protocols } = yargs(process.argv.slice(2))
   .alias("m", "models")
   .string("m")
   .describe("m", "The path to directory with aws-models.")
   .demandOption(["models"])
+  .alias("p", "protocols")
+  .string("p")
+  .describe(
+    "p",
+    "List of protocols to copy e.g. awsJson1,restXml, default all, list: \n" +
+      `awsJson1_1,
+restJson1,
+awsJson1_0,
+awsQuery,
+restXml,
+ec2Query`
+  )
   .help().argv;
 
+const getSdkId = (model) => {
+  const { shapes } = model;
+  const service = Object.values(shapes).find((shape) => shape.type === "service") || {};
+  const { traits } = service;
+  for (const [trait, value] of Object.entries(traits)) {
+    if (trait === "aws.api#service") {
+      return value.sdkId;
+    }
+  }
+  return "unknown";
+};
+
+const getWireProtocol = (model) => {
+  const { shapes } = model;
+  const service = Object.values(shapes).find((shape) => shape.type === "service") || {};
+  for (const trait of Object.keys(service.traits || {})) {
+    if (trait.startsWith("aws.protocols#")) {
+      return trait.split("aws.protocols#").pop() || "unknown";
+    }
+  }
+  return "unknown";
```

Comment:
Throw an error here?

The copy-models may continue copying with `"unknown"` value returned for wire protocol.